content/doc/book/system-administration/security.adoc

```diff
@@ -70,8 +70,8 @@ The following topics discuss other security features that are on by default. You
 
 == Disabling Security
 
-One may accidentally set up security realm / authorization in such a way that
-you may no longer able to reconfigure Jenkins.
+One may accidentally setup security realm / authorization in such a way that
```

Comment:
This was the correct usage of "set up", but this sentence could use "a" before "security realm".

Reply:
Agreed - change added